### DISCUSSION TITLE
Strong Axis vs Weak Axis

### DISCUSSION THREAD

**Discussion by mkawahara22** [FEALiTE/FEALiTE2D#21] [Q&A] [answered]
First of all, thanks for the amazing library!

Quick question that I had.  Say I have a column, but I would like to rotate the column 90 degrees for analysis.  How/where would I accomplish this?

Would I just flip the x and y properties when creating the `IFrame2DSection`?   Or is there a method somewhere to do this when I add the `FrameElement2D` to the structure?

Thanks!

**Comment by SaMohamed** [comment, marked as answer]
Thank you for passing by.
Yes rotation of axes inc2D analysis makes no sense because there is no strong axis and week axis.

As you suggested, you can flip the x and y properties of the section.
